pom.xml

```diff
@@ -127,7 +127,7 @@
     <!-- some of plugins requires JDK 11 -->
     <recommendedJavaBuildVersion>11</recommendedJavaBuildVersion>
 
-    <junitBomVersion>5.9.2</junitBomVersion>
+    <junitBomVersion>5.9.3</junitBomVersion>
```

Comment:
In parent we have the same ... to think - can be inherit?

Reply:
Not yet released Parent. We should slowly remove property overwrites with parent pom upgrades